bin/plugin/commands/changelog.js

```diff
@@ -757,6 +757,76 @@ function sortFeatureGroups( featureGroups ) {
 	);
 }
 
+/**
+ *
+ * @param {IssuesListForRepoResponseItem[]} pullRequests List of pull requests.
+ *
+ * @return {string} The formatted changelog string.
+ */
+function formatContributors( pullRequests ) {
+	let contributorsList = '';
+
+	const ftcPRs = pullRequests.filter( ( pr ) => {
+		if ( pr.user.login === 'dependabot[bot]' ) {
+			return false;
+		}
+
+		return pr.labels.find(
+			( { name } ) => name.toLowerCase() === 'first-time contributor'
+		);
+	} );
```

Comment:
When I talked about this idea with @priethor, he mentioned that sometimes users open more than one PR and since they don't have any commits all of them get this label. That's why it would be good to ensure that the list of users is unique for a given release. Maybe we need to improve the GitHub action that applies the label, to ensure it correctly removes it from the PR at the time of merging if that is a subsequent PR.

Reply:
Sounds good. Where do these actions live? I found the `workflows/` dir in the repo but I couldn't see anything about `First-time contributor`.

Reply:
How about I simply group the PRs by username? That way if the user has more than 1 PR merged in this milestone then they get the credits for that only.

Reply:
There is an npm package with some custom automations:

https://github.com/WordPress/gutenberg/tree/trunk/packages/project-management-automation

It’s definitely nice to have and grouping by username should be perfect for start like you both pointed out.

I was mostly contemplating on the current state of the automation that applies all those labels 😃

Reply:
> There is an npm package with some custom automations:
>
> https://github.com/WordPress/gutenberg/tree/trunk/packages/project-management-automation

That's super cool. Diving into that now 👀 

> It’s definitely nice to have and grouping by username should be perfect for start like you both pointed out.

If we're happy with the general direction I'll proceed with refactoring and tidying up this PR.